docs/concepts-components/concepts-components.md

```diff
@@ -9,6 +9,10 @@ The below are the building blocks of Restel.
 
 Represents a single APIs that has to be called (and tested).
 
+## Test Wrapper:
+
+Represents a wrapper for a Test API Intended to pass API specific **Restel Variables** to customise the expected response, status code of the API.
```

Comment:
Worth mentioning,

In the context of Test scenario. wrappers can be used interchangeably with `Test APIs`. We go with wrappers when we have to pass different parameters for the same Test APIs and hence performing different set of tests using the same API.